easybuild/easyconfigs/l/libgpuarray/libgpuarray-0.7.6-fosscuda-2018b-Python-3.6.6.eb

```diff
@@ -0,0 +1,48 @@
+easyblock = 'CMakeMake'
+
+name = 'libgpuarray'
+version = '0.7.6'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://deeplearning.net/software/%(name)s/'
+description = "Library to manipulate tensors on the GPU."
+
+toolchain = {'name': 'fosscuda', 'version': '2018b'}
+
+source_urls = ['https://github.com/Theano/%(name)s/releases/download/v%(version)s/']
+sources = [SOURCE_TAR_GZ]
+checksums = ['887b6433a30282cb002117da89b05812c770fd9469f93950ff3866ddd02bfc64']
+
+builddependencies = [('CMake', '3.12.1')]
+
+configopts = '-DCMAKE_BUILD_TYPE=Release'
+
+dependencies = [
+    ('Mako', '1.0.9', '-Python-%(pyver)s'),
+    ('NCCL', '2.3.7'),
+    ('Python', '3.6.6'),
```

Comment:
Preferably put Python as the first dependency since it is the main one, and Mako depends on it.